### PR TITLE
PA-2448: Enhanced user information

### PR DESCRIPTION
**Pull request by jdtoombs**
This PR includes:

- moving user logout to the header
- include agency and system role(s) of user
- move help modal down to navigation bar

newprofile

### CODE REVIEW

File: frontend/src/components/layout/AppNavBar/AppNavBar.scss [hunk 1]
```diff
@@ -58,19 +58,6 @@ nav.map-nav {
         }
       }
     }
-    &.profile {
```
Comment (jdtoombs) [inline frontend/src/components/layout/AppNavBar/AppNavBar.scss:61]: Moved with the component

File: frontend/src/components/layout/AppNavBar/AppNavBar.test.tsx [hunk 2]
```diff
@@ -332,73 +332,4 @@ describe('AppNavBar', () => {
       });
     });
   });
-
```
Comment (jdtoombs) [inline frontend/src/components/layout/AppNavBar/AppNavBar.test.tsx:335]: Moved to header tests

File: frontend/src/components/layout/Header/UserProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    width: 300px;
+    height: 120px;
+  }
+  .nav-link {
+    color: #fff;
+    padding: 1px;
+  }
+`;
+
+export const UserProfile: React.FC = () => {
```
Comment (Fosol) [inline frontend/src/components/layout/Header/UserProfile.tsx:25]: Add doc

**Review by Fosol — APPROVED** [review]

**Review by agalbrai — APPROVED** [review]
Looks good!

File: frontend/src/components/layout/Header/Header.tsx [hunk 4]
```diff
@@ -97,12 +97,12 @@ const Header = () => {
           <h1 className="shortAppName">PIMS</h1>
         </Nav.Item>
       </Nav>
+      {!history.location.pathname.includes('/login') && <UserProfile />}
```
Comment (devinleighsmith) [inline frontend/src/components/layout/Header/Header.tsx:100]: I think this is potentially fragile if we end up with more then one page where the user is not logged in. Should be able to base this off of the keycloak status.

File: frontend/src/components/layout/Header/UserProfile.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  const configuration = useConfiguration();
+  const lookupCodes = useCodeLookups();
+  const agencyOptions = lookupCodes.getByType(API.AGENCY_CODE_SET_NAME);
+  const roles = keycloak.roles.join(', ');
+
+  return (
+    <>
+      <Image src={profileUrl} rounded style={{ height: 30, width: 30 }} />
+      <StyleDropDown className="px-0" title={displayName} id="user-dropdown">
+        {history ? (
```
Comment (devinleighsmith) [inline frontend/src/components/layout/Header/UserProfile.tsx:43]: ?

File: frontend/src/components/layout/Header/UserProfile.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+            <p style={{ margin: 5 }}>
+              <b>
+                {
+                  agencyOptions.find(
+                    (x: ILookupCode) => x.id.toString() === keycloak.agencyId?.toString(),
+                  )?.name
+                }
+              </b>
+            </p>
+            <div style={{ backgroundColor: '#f2f2f2', margin: 5 }}>
```
Comment (devinleighsmith) [inline frontend/src/components/layout/Header/UserProfile.tsx:54]: I'd like to stay away from using style as much as this, it makes css inheritance impossible which makes re-use difficult. Prefer either styled components or bootstrap classes.

Comment (devinleighsmith) [reply frontend/src/components/layout/Header/UserProfile.tsx:54]: Might not need to rework this, what do you think @goemen ?

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/components/layout/Header/UserProfile.tsx (hunk 6 above)
Comment (jdtoombs) [reply frontend/src/components/layout/Header/UserProfile.tsx:54]: I can move them into styled components!

File: frontend/src/components/layout/Header/Header.tsx (hunk 4 above)
Comment (jdtoombs) [reply frontend/src/components/layout/Header/Header.tsx:100]: Will adjust

**Pull request merged**